Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#rsschool-cv\nhttps://sxtim.github.io/rsschool-cv/cv\nhttps://sxtim.github.io/rsschool-cv/\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision 41ba4d780346c84d86e23ec5ba09eac1a40ccb1b)
+++ b/README.md	(date 1640762998958)
@@ -1,3 +1,1 @@
 #rsschool-cv
-https://sxtim.github.io/rsschool-cv/cv
-https://sxtim.github.io/rsschool-cv/
